### PR TITLE
add firebase functions example

### PR DESCRIPTION
**Pull request by kemalturk**

### CODE REVIEW

File: firebase-functions/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Medium Post
+https://medium.com/@kmltrk07/how-to-deploy-gofiber-app-to-firebase-functions-8d4d537a4464
```
Comment (ReneWerner87) [inline firebase-functions/README.md:2]: can you add a bit more, a short description of the example

please also link in the root readme

File: firebase-functions/src/database/database.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	if err != nil {
+		log.Fatalf("functions.init: Database init : %v\n", err)
+	}
+
+	return db
+}
+
+func credentials() []byte {
+	// TODO: Replace with your Credentials
+	data := map[string]interface{}{
```
Comment (ReneWerner87) [inline firebase-functions/src/database/database.go:34]: add this part as an instruction in the readme.md

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]
thx, we need some little adjustments

**Comment by ReneWerner87** [comment]
@kemalturk

**Comment by kemalturk** [comment]
@ReneWerner87 I've added detailed documentation to readme. If you have any suggestions or would like to make changes, please feel free to let me know.

**Comment by ReneWerner87** [comment]
@kemalturk thx, just the link in the root README.md is missing

can you do this https://github.com/gofiber/recipes/blob/master/README.md (alphabetical sorting)

**Comment by kemalturk** [comment]
@ReneWerner87 Done! I've added the missing link to the root README.md

**Pull request merged**